registry/registry_test.go

```diff
@@ -148,6 +148,14 @@ func TestGetCipherSuite(t *testing.T) {
 	if err == nil {
 		t.Error("did not return expected error about unknown cipher suite")
 	}
+
+	var insecureCipherSuites = []string{"TLS_RSA_WITH_RC4_128_SHA", "TLS_RSA_WITH_AES_128_CBC_SHA256", "TLS_ECDHE_ECDSA_WITH_RC4_128_SHA", "TLS_ECDHE_RSA_WITH_RC4_128_SHA", "TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256", "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256"}
```

Comment:
Can we do these two things please?

1. put each item in `insecureCipherSuites` slice on a separate line (makes it easier to edit, etc.)
2. add a new line char before the `for` loop so it separates the above slice nicely from the loop -- makes it easier to read the code